packages/app-toolbox/src/Verify.tsx

```diff
@@ -48,51 +62,34 @@ class Verify extends React.PureComponent<Props, State> {
   }
 
   render () {
-    return (
-      <div className='toolbox--Verify'>
-        {this.renderInput()}
-        {this.renderAddress()}
-        {this.renderSignature()}
-      </div>
-    );
-  }
-
-  renderAddress () {
-    const { t } = this.props;
-    const { defaultPublicKey, isValidAddress } = this.state;
-
-    return (
-      <div className='ui--row'>
-        <InputAddress
-          className='full'
-          defaultValue={defaultPublicKey}
-          isError={!isValidAddress}
-          isInput
-          label={t('verify using address')}
-          onChange={this.onChangeAddress}
-        />
-      </div>
-    );
-  }
-
-  renderInput () {
     const { t } = this.props;
-    const { data, isHexData } = this.state;
+    const { cryptoOptions, cryptoType, data, isHexData } = this.state;
 
     return (
-      <>
+      <div className='toolbox--Verify'>
+        {this.renderAddress()}
         <div className='ui--row'>
           <Input
             autoFocus
             className='full'
-            label={t('using the following data (hex or string)')}
+            help={t('The data that was signed. This is used in combination with the signature for the verification. It can either be hex or s string.')}
```

Comment:
typo 
```suggestion
            help={t('The data that was signed. This is used in combination with the signature for the verification. It can either be hex or a string.')}
```